.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+name: Build
```

Comment:
Might want this name (and filename) to be `CI` or `test`, as we're doing more than just building.

Reply:
Renamed to CI.